src/views/create-dataset/CreateDataset.tsx

```diff
@@ -0,0 +1,71 @@
+// components/FormPresenter.tsx
+
+import { ChangeEvent, FormEvent } from 'react'
+import { Alert, Button, Col, Form, Row } from '@iqss/dataverse-design-system'
+import { useTranslation } from 'react-i18next'
+import styles from '/src/sections/dataset/Dataset.module.scss'
+import { SeparationLine } from '../../components/ui/SeparationLine/SeparationLine'
+import { RequiredFieldText } from '../../components/Forms/RequiredFieldText/RequiredFieldText'
+
+/*
+ * TODO:
+ * out-of-scope: Find submit action source
+ * out-of-scope: Loading state management
+ */
+
+interface CreateDatasetFormPresenterProps {
+  formData: { createDatasetTitle: string }
+  handleCreateDatasetFieldChange: (event: ChangeEvent<HTMLInputElement>) => void
+  handleCreateDatasetSubmit: (event: FormEvent<HTMLFormElement>) => void
+  submitting: boolean
+  submitComplete: boolean
+}
+export default function CreateDatasetFormPresenter({
+  handleCreateDatasetFieldChange,
+  handleCreateDatasetSubmit,
+  submitting,
+  submitComplete
+}: CreateDatasetFormPresenterProps) {
+  const { t } = useTranslation('createDataset')
+  // TODO: Probably replace this with a FormSkeleton or remove entirely
+  // if (loading) {
+  //   return <DatasetSkeleton />
+  // }
```

Comment:
In a Model - View - Presenter the decision of when to show a loader would be in the presenter, but the rendering of the loading should be placed in the view, so this commented code should be in that other component CreateDatasetContainer

But please, remove all commented code since it's a bad practice. You can leave the TODO's as long as they point to an issue to asses the TODO